### PR TITLE
Reduced cardinality of content to improve performance

### PR DESCRIPTION
**Pull request by nwrigh**
Improve performance so it won't bring down neo

### CODE REVIEW

**Comment by JuliaFernee** [comment]
+1

File: people/cypher.go [hunk 1]
```diff
@@ -88,14 +88,15 @@ func (pcw CypherDriver) Read(uuid uuid.UUID) (person Person, found bool, err err
 	results := []struct {
 		Rs []neoReadStruct
 	}{}
+	sixMonthsEpoch := time.Now().Unix() - 15552000
```
Comment (sjwells) [inline people/cypher.go]: being picky, I'd create this number by multiplying together the different parts to make it clear it represents 6 months, e.g.
time.Now().Unix() - (60 * 60 * 24 * 30 * 6)
It also makes it easier to amend to 5 months without getting it wrong, if we ever wanted to

Comment (nwrigh) [inline people/cypher.go]: Good point so done :D

**Comment by scott-ace-newton** [comment]
:+1:

**Pull request merged**